api/src/wfl/api/handlers.clj

```diff
@@ -65,10 +65,12 @@
   "Return the workflows managed by the workload."
   [request]
   (log/info (select-keys request [:request-method :uri :parameters]))
-  (let [uuid (get-in request [:path-params :uuid])]
+  (let [uuid (get-in request [:path-params :uuid])
+        {:keys [status]} (get-in request [:parameters :query])]
```

Comment:
I think this binding can be made more clear.  Would this work instead?
```
status (get-in request [:parameters :query :status]
```

https://clojuredocs.org/clojure.core/get-in

Reply:
Seems like it :)